packages/web3-eth-debug/types/index.d.ts

```diff
@@ -0,0 +1,155 @@
+    (at your option) any later version.
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file index.d.ts
+ * @author Prince Sinha <sinhaprince013@gmail.com>
+ * @date 2019
+ */
+
+import {provider} from 'web3-providers';
+import {AbstractWeb3Module, Web3ModuleOptions} from 'web3-core';
+import * as net from 'net';
+
+export class Debug extends AbstractWeb3Module {
+    constructor(provider: provider, net?: net.Socket|null, options?: Web3ModuleOptions, accounts?: any);
+
+    backTraceAt(
+        filename: string,
+        callback?: (error: Error, result: any) => void
+    ): Promise<any>;
+
+    blockProfile(
+        file: string,
+        seconds: number,
+        callback?: (error: Error, result: any) => void
+    ): Promise<any>;
+
+    cpuProfile(
+        file: string,
+        seconds: number,
+        callback?: (error: Error, resukt: string) => void
+    ): Promise<any>;
+
+    dumpBlock(
+        blockNumber: number,
+        callback?: (error: Error, result: any) => void
+    ): Promise<any>;
+
+    gcStats(
+        callback?: (error: Error, result: Stats) => void
+    ): Promise<Stats>;
+
+    getBlockRlp(
+        blockNumber: number,
+        callback?: (error: Error, result: string) => void
+    ): Promise<string>;
+
+    goTrace(
+        file: string,
+        seconds: number,
+        callback?: (error: Error, result: any) => void
+    ): Promise<any>;
+
+    memStats(callback?: (error: Error, result: any) => void): Promise<any>;
+
+    seedHash(
+        blockNumber: number,
+        callback?: (error: Error, result: string) => void
+    ): Promise<string>;
+
+    setHead(
+        blockNumber: number,
+        callback?: (error: Error, result: any) => void
+    ): Promise<any>;
+
+    setBlockProfileRate(
+        rate: number,
+        callback?: (error: Error, result: any) => void
+    ): Promise<any>;
+
+    stacks(callback?: (error: Error, result: any) => void): Promise<any>;
+
+    startCPUProfile(
+        file: string,
+        callback?: (error: Error, result: any) => void
+    ): Promise<any>;
+
+    startGoTrace(
+        file: string,
+        callback?: (error: Error, result: any) => void
+    ): Promise<any>;
+
+    stopCPUProfile(callback?: (error: Error, result: any) => void): Promise<any>;
+
+    stopGoTrace(callback?: (error: Error, result: any) => void): Promise<any>;
+
+    traceBlock(
+        blockRlp: string,
+        options?: {},
+        callback?: (error: Error, result: any) => void
+    ): Promise<any>;
+
+    traceBlockByNumber(
+        blockNumber: number,
+        options?: {},
```

Comment:
please define this if you do not know do `any` `{}` will not let a `ts` developer pass in a dynamic options object here.